### PR TITLE
Feature/setup router

### PR DESCRIPTION
**Pull request by Josephhaefling**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description
We set up React Router so that the user can move between different pages. We also adjusted the fetch to GET more painting images.

### Why is this change required? What problem does it solve?
This change allows the user to click on a painting of an image and see more info about the painting

### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
n/a
### Files modified:
- [x] App.js
- [x] PaintingPage.js

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -72,10 +75,19 @@ function App() {
...
+  return (
+  <>
+    <Route exact path= '/' render={() => mainPage} />
+    <Route exact path= '/:paintingTitle' render={({match}) => {
+      const {id} = match.params
+      return <PaintingInfo paintingInfo={selected} />
+    
+    }} />
+  </> )
 }
```
Comment (carflor) [inline src/App/App.js:92]: Great start to the Router

File: src/Gallery/Gallery.css [hunk 2]
```diff
@@ -2,6 +2,10 @@
   max-height: 40vh;
   max-width: fit-content;
  }
+
+ .hidden {
+   display: none;
+ }
```
Comment (carflor) [inline src/Gallery/Gallery.css:8]: Nifty.

File: src/Painting/Painting.js [hunk 3]
```diff
@@ -4,10 +4,15 @@ import './Painting.css';
 
 function Painting (props) {
 
+  const hideBrokenImages = () => {
+    const img = document.getElementById(props.painting.contentId)
+    img.classList.add('hidden')
+  }
+
```
Comment (carflor) [inline src/Painting/Painting.js:11]: This is a clean fix. Good job Charles.

**Comment by cwefso** [comment]
Cool, I think this puts us in a really good spot

**Pull request merged**